examples/datasource-basic/cypress/integration/config.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { e2e } from '@grafana/e2e';
+
+const configEditor = e2e.getSelectors({
+    timeField: 'Macro default time field',
+    apiKey: 'Secret api key',
+});
```

Comment:
Nice but I would expect this to be used in the production code too, not just in the test code. I think there's were our thin layer upon Cypress shines.

Reply:
I will update!